app/models/stripe_wrapper.rb

```diff
@@ -0,0 +1,28 @@
+module StripeWrapper
+  class Charge
+    attr_reader :response, :error_message
+
+    def initialize(options={})
+      @response = options[:response]
+      @error_message = options[:error_message]
```

Comment:
You could align these as well (same way as you align parameters in `self.create` method.